openedx/core/djangoapps/content/block_structure/models.py

```diff
@@ -0,0 +1,104 @@
+"""
+Models used by the block structure framework.
+"""
+
+from django.core.files.base import ContentFile
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+from openedx.core.djangoapps.xmodule_django.models import UsageKeyField
+from openedx.core.lib.block_structure.exceptions import BlockStructureNotFound
+
+
+def upload_to_path(bs_model, filename):  # pylint:disable=W0613
+    """
+    Returns path name to use for the given
+    BlockStructureModel instance.
+    """
+    return unicode(bs_model.data_usage_key)
+
+
+class BlockStructureModel(TimeStampedModel):
+    """
+    Model for storing Block Structure information.
+    """
+    class Meta(object):
+        db_table = 'block_structure'
+
+    data_usage_key = UsageKeyField(
+        u'Identifier of the data being collected.',
+        blank=False,
+        max_length=255,
+        db_index=True,
+    )
+    data_version = models.CharField(
+        u'Version of the data at the time of collection.',
+        blank=True,
+        max_length=255,
+    )
+    data_edit_timestamp = models.DateTimeField(
+        u'Edit timestamp of the data at the time of collection.',
+        blank=True,
+        null=True,
+    )
+
+    transformers_schema_version = models.CharField(
+        u'Representation of the schema version of the transformers used during collection.',
+        blank=False,
+        max_length=255,
+    )
+    block_structure_schema_version = models.CharField(
+        u'Version of the block structure schema at the time of collection.',
+        blank=False,
+        max_length=255,
+    )
+
```

Comment:
Same here. Keep the spacing the same between all fields definitions if possible.